.github/workflows/ci.yaml

```diff
@@ -141,7 +141,7 @@ jobs:
     - name: Cache Python and its deps
       uses: actions/cache@v2.1.6
       with:
-        key: ${{ runner.os }}-python-${{ matrix.python-version }}-${{ hashFiles('setup.py', 'requirements/*.txt') }}
+        key: ${{ runner.os }}-python-deps-${{ matrix.python-version }}-${{ hashFiles('setup.py', 'requirements/*.txt') }}
```

Comment:
do we need it in the end?

Reply:
seems like we do, because windows tests fail with old cache key

Reply:
seems like the issue is resolved in 4c4ef4d , lets remove that change